.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ commands:
             description: the CircleCI token to execute a CircleCI job on the smoke test repo
         remote_branch:
             type: string
-            default: main
+            default: master
```

Comment:
@tripodsan the helix pages default is still `master`

Reply:
right. sorry.